network.go

```diff
@@ -521,6 +527,13 @@ func NetworkOptionDeferIPv6Alloc(enable bool) NetworkOption {
 	}
 }
 
+// NetworkOptionDefaultGW returns an option setter to set if a network needs default gateway
+func NetworkOptionDefaultGW(defaultGW bool) NetworkOption {
```

Comment:
I think I may have mentioned it a few times at DockerCon EU :grinning: 
And +1 on calling it "Isolated"